lib/input-selection.js

```diff
@@ -17,7 +17,13 @@ function set(element, start, end) {
   // Some browsers explode if you use setSelectionRange
   // on a non-focused element
   if (document.activeElement === element && element.setSelectionRange) {
-    element.setSelectionRange(start, end);
+    // Some Android Chrome keyboards need to
```

Comment:
Sounds correct to me, although it seems odd that some Samsung keyboards behave different from other Samsung keyboards